playbooks/roles/ocp-config/README.md

```diff
@@ -53,6 +70,39 @@ chronyconfig:
       - 10.1.2.3/16
 ```
 
+*Machine Config for Luks Encryption example*
```

Comment:
Instead of showing MachiineConfig, it will help to have Ansible variable example here.

Reply:
Added Ansible variable example instead of Machine Config.